deploy/examples/object-a.yaml

```diff
@@ -0,0 +1,78 @@
+#################################################################################################################
+# Create an object store with settings for shared pools in a production environment. A minimum of 3 hosts with
+# OSDs are required in this example. This example shows two object stores being created with the same
+# shared metadata and data pools. The pool sharing will utilize RADOS namespaces to keep the object store
+# data independent, while avoiding the growth of PGs in the cluster.
+#  kubectl create -f object-shared-pools.yaml
+#  kubectl create -f object-a.yaml -f object-b.yaml
+#################################################################################################################
+apiVersion: ceph.rook.io/v1
```

Comment:
there are a lot of yamls and it is confusing, can we sprate this all in a new folder named share-object, which will help in differentiating among the normal install which is commany used

Reply:
I hesitate to create other subfolders for each scenario. IMO it's best to have a single folder unless a scenario has many manifests. This scenario has three example manifests. Let me think about it...

Reply:
I'm thinking let's not create a subdirectory for now at least